strapiExperiment/.gitignore

```diff
@@ -0,0 +1 @@
+.env
```

Comment:
this doesn't seem like an exhaustive list. https://github.com/bcgov/devhub-app-web/blob/master/app-web/.gitignore

Reply:
nvm, just didn't know the structure